docs/intro.md

```diff
@@ -5,25 +5,37 @@ title: Introduction
 
 ## What is Jitsi?
 
-Jitsi is a [collection of Open Source projects](architecture.md) which provide state-of-the-art video conferencing
+Welcome to Jitsi! A [collection of Open Source projects](architecture.md) which provide state-of-the-art video conferencing
 capabilities that are secure, easy to use and easy to self-host.
 
+In plain simple words, Jitsi allows you to interact with other people though video and audio using your browser, your phone or a desktop app, and brings you all kind of related useful functionalities like text chat, screen sharing, and video recording. All of this wrapped in a friendly interface and without forgetting your security.
+
+
 ## About this handbook
 
-This handbook aims to be the one-stop-shop for all Jitsi documentation.
+This handbook aims to be the one-stop-shop for all Jitsi documentation. 
+
+Whether you are a developer seeking to integrate Jitsi in one of your apps, a more experienced user looking for a specific configuration setup, or even if you are newcomer and have not experience with Jitsi at all, this Handbook will try to help you get there. 
+
+:::note this is a work in progress.
+Notice that some topics or sections may be vague or not as accurate as you may need. We try very hard to keep changes reflected in the docs, but overall the Jitsi Docs Project it's still quite young. 
 
-:::note It's work in progress.
-If you want to help out please create a **Pull Request** in our [Github Repo](https://github.com/jitsi/handbook)!
+- If you want to help out please consider creating a **Pull Request** in our [Github Repo](https://github.com/jitsi/handbook). No contribution is small!
+
+- Not found what you wanted? Post your question in the [Jitsi Comunity Forums](https://community.jitsi.org/).
 :::
 
-The content is divided in 3 main areas:
+That being said... lets go!
```

Comment:
To be honest, I am biased because I wrote the sentence `The content is divided in 3 main areas:`,
but still I think it should be kept this short.
People know that the next part after this is a description of sections, so no need for long words imo.